__tests__/dataframe.test.ts

```diff
@@ -1504,8 +1503,9 @@ describe("join", () => {
     const expected = pl.DataFrame({
       foo: [1, 2, 3, null],
       bar: [6, 7, 8, null],
-      ham: ["a", "b", "c", "d"],
+      ham: ["a", "b", "c", null],
       apple: ["x", null, null, "y"],
+      ham_right: ["a", null, null, "d"],
```

Comment:
why did this change?

Reply:
It's due to this change: [Preserve left and right join keys in outer joins](https://github.com/pola-rs/polars/blob/main/docs/releases/upgrade/0.20.md#preserve-left-and-right-join-keys-in-outer-joins)

Reply:
I think this syntax has [changed](https://github.com/pola-rs/polars/pull/16417) today. :)